install/share/ipa.conf.template

```diff
@@ -47,7 +47,7 @@ FileETag None
 
 # FIXME: WSGISocketPrefix is a server-scope directive.  The mod_wsgi package
 # should really be fixed by adding this its /etc/httpd/conf.d/wsgi.conf:
-WSGISocketPrefix /run/httpd/wsgi
+WSGISocketPrefix $WSGI_PREFIX_DIR
```

Comment:
```$WSGI_PREFIX_DIR``` is fine. The syntax ```${WSGI_PREFIX_DIR}``` is only required when the variable is surrounded by alphanumeric chars.